src/plugin.js

```diff
@@ -91,12 +91,17 @@ export default function ({ types: t }) {
         // TODO add other webpack checks
         if (config.module.loaders.some((l) => l.test.test(args[0].value))) {
           const [{ value: filePath }] = args;
+          const isRelativeFilePath = /^\.\.?[\/]/.test(filePath);
 
-          const fileAbsPath = resolve.sync(
+          const resolver = ResolverFactory.createResolver({
+            fileSystem: new SyncNodeJsInputFileSystem(),
+            ...config.resolve,
+          });
+
+          const fileAbsPath = resolver.resolveSync(
+            {},
+            isRelativeFilePath ? dirname(filenameRelative) : process.cwd(),
```

Comment:
What the problem to always use `dirname(filenameRelative)`, why we need `process.cwd`?